shell/common/picture_serializer.cc

```diff
@@ -19,8 +19,9 @@ bool PngPixelSerializer::onUseEncodedData(const void*, size_t) {
 SkData* PngPixelSerializer::onEncode(const SkPixmap& pixmap) {
   SkDynamicMemoryWStream stream;
 
-  bool encode_result = SkEncodeImage(
-      &stream, pixmap, SkEncodedImageFormat::kPNG, 80 /* quality */);
+  SkPngEncoder::Options options;
+  options.fUnpremulBehavior = SkTransferFunctionBehavior::kRespect;
+  bool encode_result = SkPngEncoder::Encode(&stream, pixmap, options);
```

Comment:
New png encode API supports embedding color space profiles and color correct unpremultiplication.  Note the that the "quality" param is always ignored for png (png is lossless).